_maps/shuttles/emergency/emergency_luxury.dmm

```diff
@@ -260,7 +260,7 @@
 /area/shuttle/escape/luxury)
 "aX" = (
 /obj/structure/table/wood/fancy,
-/obj/item/reagent_containers/food/snacks/customizable/pizza,
+/obj/item/food/pizza/dank,
```

Comment:
dank pizza cringe

Reply:
I think it should be fine for the time being.

Reply:
> dank pizza cringe

its temporary